drdos/funcs.fdo

```diff
@@ -526,8 +526,8 @@ chdir80:				; arrive here when root reached
 	push 	ss
 	pop 	ds
 	mov	si,offset pri_pathname	; DS:SI -> new LDT_ we just built
-	mov	cx,LDT_LEN
-	rep	movsb			; copy the new LDT_ into place
+	mov	cx,LDT_LEN / 2  ; LDT_LEN is 58h (88 bytes) so copy in words

```

Comment:
This is dangerous, as it is not given that LDT_LEN will forever be a multiple of two!!!

Reply:
Maybe we can state when declaring LDT_LEN equ	58h that is should be multiple of 2 (even).
If not, we can try to play with conditional assembly such as:

IF LDT_LEN AND 1
	mov	cx,LDT_LEN
	rep	movsb
ELSE
	mov	cx,LDT_LEN / 2
	rep	movsw
ENDIF

Reply:
I opt for the conditional, as it is safer. Do you want to implement this? Otherwise I can merge as is and apply the change by myself.

Reply:
If you feel confident I would prefer you to apply the conditional. I am not quite familiar with NASM and JWASM macros.